views/surveys/results.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+global $db;
+
+// Prevent navigating to page through URL
+if ($_SERVER['REQUEST_METHOD'] == 'GET') {
+    header("Location: /surveys");
+    die();
+}
+
+$class = html_entity_decode($_POST['classes'], ENT_COMPAT);
+
+# Display Page
+include ('header.php');
+
+?>
+
+<script>
+    function copyToClipboard(num) {
+        var tempInput = document.createElement("textarea");
+        tempInput.style = "position: absolute; left: -1000px; top: -1000px";
+        tempInput.value = document.querySelector("#testCopy" + num).innerText;
+        console.log(tempInput.value);
+        document.body.appendChild(tempInput);
+        tempInput.select();
+        document.execCommand("copy");
+        document.body.removeChild(tempInput);
+        
+        alert("Individual has been copied successfully!");
+        //var dummyContent = document.querySelector("#testCopy" + num).innerText;
+        
+        
+        //var dummy = $('<input>').val(dummyContent).appendTo('body').select();
+        //document.execCommand('Copy', false, null);
+        //console.log(dummyContent);
+    }    
+ 
+    function copyClassToClipboard() {
+     
+        var tempInput = document.createElement("textarea");
+        tempInput.style = "position: absolute; left: -1000px; top: -1000px";
+        tempInput.value = document.querySelector("#entireClass").innerText.replace("Back", "");
+        console.log(tempInput.value);
+        document.body.appendChild(tempInput);
+        tempInput.select();
+        document.execCommand("copy");
+        document.body.removeChild(tempInput);
+        
+        
+        //var dummyContent = document.querySelector("#testCopy" + num).innerText;
+        
+        
+        //var dummy = $('<input>').val(dummyContent).appendTo('body').select();
+        //document.execCommand('Copy', false, null);
+        //console.log(dummyContent);
+    }    
+      
+    function expand() {
+        var originalContents = document.body.innerHTML;
+        $('.collapse').collapse('toggle');    
+        setTimeout(copyClassToClipboard, 200);
+        setTimeout(function(){document.body.innerHTML = originalContents;}, 300);
+        setTimeout(function(){
+            alert("Class has been copied successfully!")
+        }, 1000);
+    }
+        
+    function collapseCards() {
+        $('.collapse').collapse('toggle');
+    }
+    
+    function downloadPDFClass(){
+        var originalContents = document.body.innerHTML;
+        $('.collapse').collapse('toggle');    
+        setTimeout(function(){
```

Comment:
I'm not entirely sure what's going on here, but it seems like for 200 ms you are replacing the contents of the page with the contents you want to print then swapping it back afterwards. There's definitely an easier way. See `window.print()` combined with [@media print](https://github.com/Capping-CPCA/Web-App/blob/d247be695cfbad73873b996e2e8b3fa4adb0e80f/views/reports/half_year.php#L214) css